gradlew

```diff
@@ -1,18 +1,20 @@
 #!/usr/bin/env sh
 
-#Copyright 2019 Esri
 #
-#Licensed under the Apache License, Version 2.0 (the "License"); you may not
-#use this file except in compliance with the License. You may obtain a copy
-#of the License at
+# Copyright 2015 the original author or authors.
```

Comment:
i guess this is generated everytime it's run but I wonder why it changed to 2015!